.gitignore

```diff
@@ -14,3 +14,4 @@
 !var/SymfonyRequirements.php
 /vendor/
 /web/bundles/
+/nbproject/private/
```

Comment:
W projektach Open-Source lepiej takie rzeczy wpisywać w ``.git/info/exclude``. Jest to taki plik `.gitignore`, ale który jest tylko na twoim komputerze.  Wiadomo, że po czasie jak projekt będzie duży to będzie multum IDE i ciągle dorastać nowe, więc lepiej tego nie trzymać.